sdk/resourcemanager/Azure.ResourceManager.Core/src/SubscriptionOperations.cs

```diff
@@ -100,22 +97,154 @@ public virtual LocationContainer GetLocations()
         /// <inheritdoc/>
         public override ArmResponse<Subscription> Get(CancellationToken cancellationToken = default)
         {
-            return new PhArmResponse<Subscription, ResourceManager.Resources.Models.Subscription>(
-                SubscriptionsClient.Get(Id.Name, cancellationToken),
+            return new PhArmResponse<Subscription, SubscriptionData>(
+                SubscriptionsRestOperations.Get(Id.Name, cancellationToken),
                 Converter());
         }
 
         /// <inheritdoc/>
         public override async Task<ArmResponse<Subscription>> GetAsync(CancellationToken cancellationToken = default)
         {
-            return new PhArmResponse<Subscription, ResourceManager.Resources.Models.Subscription>(
-                await SubscriptionsClient.GetAsync(Id.Name, cancellationToken).ConfigureAwait(false),
+            return new PhArmResponse<Subscription, SubscriptionData>(
```

Comment:
each of these methods need the scope boilerplate code on it like exists in Azure.ResourceManager.Resources

Reply:
Done